conf/application.conf

```diff
@@ -7,8 +7,8 @@ dbinfo = {
   properties = {
     driver = "com.mysql.cj.jdbc.Driver"
     url = "jdbc:mysql://localhost/oversitedb?serverTimezone=UTC"
-    user = "root"
-    password = "MySQL2468012345"
+    user = "myuser"
+    password = "myuser"
```

Comment:
Having passwords in the repo is not a great idea.

Reply:
Then how do we do this?

Reply:
Leave it blank. Locally use an application.conf override that isn't in the repo.